### ISSUE TITLE
Add queries for fee assets

### ISSUE THREAD

**Issue by dudo50** [paraspell/xcm-tools#531]
Add a query for fee assets available in a specific chain. This will help developers choose a fee asset (and give them ability to give users a choice to select fee asset they preffer).

**Comment by dudo50** [comment]
Postponed

**Comment by bkontur** [comment]
@franciscoaguirre
there is also a runtime API:

```
impl xcm_runtime_apis::fees::XcmPaymentApi<Block> for Runtime {
		fn query_acceptable_payment_assets(xcm_version: xcm::Version) -> Result<Vec<VersionedAssetId>, XcmPaymentApiError> {

		fn query_delivery_fees(destination: VersionedLocation, message: VersionedXcm<()>) -> Result<VersionedAssets, XcmPaymentApiError> {
```

**Comment by dudo50** [comment]
@bkontur thanks,

We are aware of this one. However, it has not yet been implemented in many chains. Very few have it, making it difficult for us to implement it universally.

With kind regards,
Team ParaSpell✨

**Comment by bkontur** [comment]
yes, that is the point, we added this runtime api to the all system parachains, and the runtime api is part of polkadot-sdk,
so anychain can use it.

If you prepare some tool that expects this API, then you can ask other parachains to add it.

**Comment by dudo50** [comment]
Thanks for the heads up, we will think of a way to check whether Parachain has implemented the API to enhance it in these queries. It is in plan for sure.

With kind regards,
Team ParaSpell✨
